lib/synapse_pay_rest/models/node/base_node.rb

```diff
@@ -353,6 +384,16 @@ def find_subnet(id:)
       Subnet.find(node: self, id: id)
     end
 
+    # Get statement for node
+    # Only available for native synapse nodes
+    # 
+    # @raise [SynapsePayRest::Error]
+    # 
+    # @return [SynapsePayRest::Statement]
+    def get_statement()
+      Statement.by_node(client: self.user.client, node: self)
+    end
+
```

Comment:
Ah, this statement endpoint would actually be useful. I hand-rolled a call to this [in the portal](https://github.com/wunderteam/portal/blob/1038212a8a1a5f7e2ad3f467eb141106cbdf177c/app/adapters/synapse/api/node.rb#L19-L22).